### DISCUSSION TITLE
Gallery source code?

### DISCUSSION THREAD

**Discussion by plettb** [CommunityToolkit/Windows#467] [Q&A] [answered]
I'm feeling a little foolish for asking this because it seems to me it's likely obvious, but where do I find the source code for the "Windows Community Toolkit Gallery" found in the store?  I've cloned the repository at https://github.com/CommunityToolkit/Windows and searched the other repositories and I still haven't found it.

**Comment by michael-hawker** [comment, marked as answer]
No problem, the sample gallery code is part of our Tooling module as we share our infrastructure between the main repo here and the Labs-Windows repo as well.

The source code for the Sample App is part of that here: https://github.com/CommunityToolkit/Tooling-Windows-Submodule/tree/main/CommunityToolkit.App.Shared

If you checked out this repo following the instructions in the readme to recurse sub-modules, you should be able to see this code when you open other solutions as well.
